modules/ROOT/pages/8.0-release-notes.adoc

```diff
@@ -119,7 +120,30 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[changes]]
 == Changes
 
-{productname} {release-version} also includes the following change<s>:
+{productname} {release-version} includes the following changes:
+
+=== Enhanced license key system with T8LK prefix
+
+The license key system in {productname} 8 has been enhanced to provide improved subscription access control and streamlined renewal processes. All commercial license keys for self-hosted deployments now require the `T8LK:` prefix to indicate version 8 compatibility.
```

Comment:
If I were a user I'd assume this means I could just update the prefix on my TinyMCE 7 license to T8LK.